### PR TITLE
Migrate to androidx

### LINKED ISSUE #79: Support Android X
And release as Laevatein `3.0.0`

### PR DESCRIPTION
**Pull request by su-makoto**
close #79

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (su-makoto) [inline build.gradle]: `Refactor > Migrate to AndoridX` needs AS 3.2 or later

File: sample/build.gradle [hunk 2]
```diff
@@ -18,14 +18,13 @@ android {
...
 }
 
-ext.supportLibraryVersion = '27.1.0'
-
 dependencies {
-    implementation "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'com.google.android.material:material:1.1.0-alpha03'
```
Comment (su-makoto) [inline sample/build.gradle]: Required`com.google.android.material:material` for resolve dependencie with `androidx.legacy:legacy-support-v4`.

**Comment by su-makoto** [comment]
@hiroyuki-seto @mrtry 
Please review this PR.
Thank you 😘

**Comment by mrtry** [comment]
👀

File: laevatein/build.gradle [hunk 3]
```diff
@@ -17,15 +17,13 @@ android {
...
-
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
-    implementation "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    implementation "com.android.support:support-annotations:${supportLibraryVersion}"
-    implementation "com.android.support:design:${supportLibraryVersion}"
-    implementation "com.android.support:exifinterface:${supportLibraryVersion}"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
```
Comment (mrtry) [inline laevatein/build.gradle]: Do you have a reason to specify alpha version?
I think that to use the stable version (1.0.2).
see: https://mvnrepository.com/artifact/androidx.appcompat/appcompat

Comment (su-makoto) [reply laevatein/build.gradle]: Ah, thank you good comment 😘 
I fix to use stable version.

> Do you have a reason to specify alpha version?

Auto specified alpha versionI , when used `Refactor > Migrate to AndroidX...` on Android Studio.
I was not aware of this, sorry.

Comment (su-makoto) [reply laevatein/build.gradle]: @mrtry I fixed this on https://github.com/nohana/Laevatein/pull/86/commits/5a05f6808f921511739954234d1cbd0237ab6e55 .
Please check again, thank you 😄

**Comment by mrtry** [comment]
LGTM 👍 👍 👍 👍

**Pull request merged**